### ISSUE TITLE
Support CSS Modules

### ISSUE THREAD

**Issue by munierujp** [future-architect/eslint-plugin-vue-scoped-css#28]
Support [CSS Modules](https://vue-loader.vuejs.org/guide/css-modules.html).

### vue-scoped-css/require-module
Add `vue-scoped-css/require-module` rule.

```vue
<template>
</template>

<!-- ✗ BAD -->
<style>
</style>

<!-- ✓ GOOD -->
<style module>
</style>
```

### vue-scoped-css/require-scoped-or-module
Add `vue-scoped-css/require-scoped-or-module` rule.

```vue
<template>
</template>

<!-- ✗ BAD -->
<style>
</style>

<!-- ✓ GOOD -->
<style scoped>
</style>

<!-- ✓ GOOD -->
<style module>
</style>
```

**Comment by ota-meshi** [comment]
@munierujp Thank you for posting this issue.

I like your proposed rules.
However, since this ESLint plugin focuses on Scoped CSS, I don't want to add any CSS Modules rules to this ESLint plugin.

**Comment by munierujp** [comment]
I see.
Thank you😄
